packages/components/src/FormField/FormFieldWrapper.tsx

```diff
@@ -124,97 +89,123 @@ export function FormFieldWrapper({
         data-testid="Form-Field-Wrapper"
         ref={wrapperRef}
       >
-        <div className={styles.horizontalWrapper}>
-          {prefix?.icon && <AffixIcon {...prefix} size={size} />}
-          <div className={styles.inputWrapper}>
-            {placeholder && (
-              <label
-                className={styles.label}
-                htmlFor={identifier}
-                style={
-                  prefixRef?.current || suffixRef?.current
-                    ? labelStyle
-                    : undefined
-                }
-              >
-                {placeholder}
-              </label>
-            )}
-
-            {prefix?.label && <AffixLabel {...prefix} labelRef={prefixRef} />}
-
-            <div className={styles.childrenWrapper} tabIndex={-1}>
-              {children}
-            </div>
-
-            {suffix?.label && (
-              <AffixLabel {...suffix} labelRef={suffixRef} variation="suffix" />
-            )}
-          </div>
-          {showClear && <ClearAction onClick={onClear} />}
-          {suffix?.icon && (
-            <AffixIcon {...suffix} variation="suffix" size={size} />
-          )}
-        </div>
-        <AnimatePresence
-          initial={toolbarVisibility === "always" ? false : true}
-        >
-          {isToolbarVisible && (
-            <motion.div
-              key="toolbar"
-              initial={toolbarAnimationEnd}
-              animate={toolbarAnimationStart}
-              exit={toolbarAnimationEnd}
-              transition={{
-                duration: tokens["timing-base"] / 1000,
-                ease: "easeInOut",
-              }}
-              tabIndex={-1}
-            >
-              <div
-                className={styles.toolbar}
-                data-testid="ATL-InputText-Toolbar"
-              >
-                {toolbar}
-              </div>
-            </motion.div>
-          )}
-        </AnimatePresence>
-      </div>
-      {description && !inline && (
-        <FormFieldDescription
-          id={descriptionIdentifier}
-          description={description}
+        <FormFieldInputHorizontalWrapper>
+          <AffixIcon {...prefix} size={size} />
+          <FormFieldInputWrapperStyles>
+            <FormFieldLabel
+              placeholder={placeholder}
+              identifier={identifier}
+              style={
+                prefixRef?.current || suffixRef?.current
```

Comment:
This is the only part of this I'm not huge on, we can probably pull that into one of these hooks, but it's not a big enough deal to hold up this particular train. Let's keep rolling!